cellprofiler/settings.py

```diff
@@ -722,6 +722,25 @@ def set_value_text(self, value_text):
             self.__default = self.str_to_value(value_text)
         except:
             logger.debug("Number set to illegal value: %s" % value_text)
+            
+    def set_min_value(self, minval):
```

Comment:
`maximum` and `minimum` are more descriptive names than `max_value` and `min_value`